src/energyplus/Test/ZoneMixing_GTest.cpp

```diff
@@ -0,0 +1,500 @@
+      EXPECT_TRUE(zm.isEmpty(ZoneMixingFields::MinimumSourceTemperatureScheduleName));
+      EXPECT_TRUE(zm.isEmpty(ZoneMixingFields::MaximumSourceTemperatureScheduleName));
+      EXPECT_TRUE(zm.isEmpty(ZoneMixingFields::MinimumOutdoorTemperatureScheduleName));
+      EXPECT_TRUE(zm.isEmpty(ZoneMixingFields::MaximumOutdoorTemperatureScheduleName));
+      if (name == zmAbs.nameString()) {
+        EXPECT_EQ(z2.nameString(), receiving_->nameString());
+        EXPECT_EQ("Flow/Zone", zm.getString(ZoneMixingFields::DesignFlowRateCalculationMethod).get());
+        EXPECT_EQ(m3sAbs, zm.getDouble(ZoneMixingFields::DesignFlowRate).get());
+        EXPECT_TRUE(zm.isEmpty(ZoneMixingFields::FlowRateperFloorArea));
+        EXPECT_TRUE(zm.isEmpty(ZoneMixingFields::FlowRateperPerson));
+        EXPECT_TRUE(zm.isEmpty(ZoneMixingFields::AirChangesperHour));
+        EXPECT_EQ(z1.nameString(), source_->nameString());
+        EXPECT_EQ(0.5, zm.getDouble(ZoneMixingFields::DeltaTemperature).get());
+      } else if (name == zmFloor.nameString()) {
+        EXPECT_EQ(z1.nameString(), receiving_->nameString());
+        EXPECT_EQ("Flow/Zone", zm.getString(ZoneMixingFields::DesignFlowRateCalculationMethod).get());
+        EXPECT_EQ(equivm3s_Floor, zm.getDouble(ZoneMixingFields::DesignFlowRate).get());
+        EXPECT_TRUE(zm.isEmpty(ZoneMixingFields::FlowRateperFloorArea));
+        EXPECT_TRUE(zm.isEmpty(ZoneMixingFields::FlowRateperPerson));
+        EXPECT_TRUE(zm.isEmpty(ZoneMixingFields::AirChangesperHour));
+        EXPECT_EQ(z2.nameString(), source_->nameString());
+        EXPECT_TRUE(zm.isEmpty(ZoneMixingFields::DeltaTemperature));
+      } else if (name == zmPerson.nameString()) {
+        EXPECT_EQ(z2.nameString(), receiving_->nameString());
+        EXPECT_EQ("Flow/Zone", zm.getString(ZoneMixingFields::DesignFlowRateCalculationMethod).get());
+        EXPECT_EQ(equivm3s_Person, zm.getDouble(ZoneMixingFields::DesignFlowRate).get());
+        EXPECT_TRUE(zm.isEmpty(ZoneMixingFields::FlowRateperFloorArea));
+        EXPECT_TRUE(zm.isEmpty(ZoneMixingFields::FlowRateperPerson));
+        EXPECT_TRUE(zm.isEmpty(ZoneMixingFields::AirChangesperHour));
+        EXPECT_EQ(z1.nameString(), source_->nameString());
+        EXPECT_TRUE(zm.isEmpty(ZoneMixingFields::DeltaTemperature));
+      } else if (name == zmACH.nameString()) {
+        EXPECT_EQ(z1.nameString(), receiving_->nameString());
+        EXPECT_EQ("Flow/Zone", zm.getString(ZoneMixingFields::DesignFlowRateCalculationMethod).get());
+        EXPECT_EQ(equivm3s_ACH, zm.getDouble(ZoneMixingFields::DesignFlowRate).get());
+        EXPECT_TRUE(zm.isEmpty(ZoneMixingFields::FlowRateperFloorArea));
+        EXPECT_TRUE(zm.isEmpty(ZoneMixingFields::FlowRateperPerson));
+        EXPECT_TRUE(zm.isEmpty(ZoneMixingFields::AirChangesperHour));
+        EXPECT_EQ(z2.nameString(), source_->nameString());
+        EXPECT_TRUE(zm.isEmpty(ZoneMixingFields::DeltaTemperature));
+      }
+    }
+  }
+}
```

Comment:
FT Tests when different zone.
* Test all combinations of Spaces one
* Test one zone one
* Do that when excluding spaces and when including them